consensus_decentralization/helper.py

```diff
@@ -361,3 +361,58 @@ def get_date_from_block(block, level='day'):
     elif level == 'day':
         return timestamp[:10]
     raise ValueError(f'Invalid level: {level}')
+
+
+def get_granularity():
+    """
+    Retrieves the granularity to be used in the analysis
+    :returns: string in ['day', 'week', 'month', 'year'] that represents the chosen granularity
+    or None if the relevant field is empty in the config file
+    :raises ValueError: if the granularity field is missing from the config file or if
+    the chosen value is not one of the allowed ones
+    """
+    try:
+        granularity = get_config_data()['granularity']
+        if granularity:
+            if granularity in ['day', 'week', 'month', 'year']:

```

Comment:
Not a big deal, but if we inlude None in this list we wouldn't need the outside if

Reply:
Also, before we used "all" instead of empty / None. I'm fine to keep the empty one but then some more docstring needs to be changed where it shows "all" as an option. Additionally, since "aggregate_by" was renamed to "granularity", it should be renamed everywhere (there are some helper functions that use it and haven't been renamed from what I can tell)